tasks/update-ocp-tag/update-ocp-tag.yaml

```diff
@@ -4,14 +4,14 @@ kind: Task
 metadata:
   name: update-ocp-tag
   labels:
-    app.kubernetes.io/version: "1.4.0"
+    app.kubernetes.io/version: "1.4.1"
   annotations:
     tekton.dev/pipelines.minVersion: "0.12.1"
     tekton.dev/tags: release
 spec:
   description: >-
-    Tekton task to update pull-spec tag with
-    valid OCP version from get-ocp-version task.
+    Tekton task to update pull-spec tag with the OCP version gotten from the get-ocp-version task. The replacement only
```

Comment:
In theory the README would be updated the same way but not too big of a deal. Feel free to resolve this without changing if you'd like

Reply:
I can fix it NP. This saves me to type `/retest` once again..

Reply:
fixed!